package.json

```diff
@@ -2,12 +2,16 @@
   "name": "handle-path",
   "version": "1.0.1",
   "description": "A simple route handling tool. Mapping a URL to a handler",
-  "main": "src/index.js",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "test": "xo && ava"
+    "lint": "eslint --ext .ts src",
+    "transpile": "tsc",
+    "unit": "ava",
+    "test": "run-s lint transpile unit"
```

Comment:
`run-s` is from [npm-run-all](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-s.md).